### DISCUSSION TITLE
cannot change mount location in mac?

### DISCUSSION THREAD

**Discussion by SyedMuhamadYasir** [iterate-ch/cyberduck#15928] [Q&A]
hi, using MD 4.5.0 (17823)

i cannot seem to change the mount location, it is stuck on the 'volumes' folder

![image](https://github.com/iterate-ch/cyberduck/assets/30082863/6909eb4b-a10a-44e8-8b2f-e82d457bdcfb)

please help!

**Comment by tatze96** [comment]
You can change the [mount location](https://docs.cyberduck.io/mountainduck/preferences/#mount-location) to a writable location of your choice. Ensure to click on `Choose...` within the dropdown menu to select a destination instead of choosing one of the proposed options to change the mount location successfully.
